client/src/index.tsx

```diff
@@ -1,9 +1,9 @@
 // Hack to ensure jQuery is registered before bootstrap.
 // See https://stackoverflow.com/questions/34120250/error-using-bootstrap-jquery-packages-in-es6-with-browserify.
 // TODO(thenuge): Probably just use Sass instead.
```

Comment:
Nit: maybe move this down as well? Can we stop doing this yet?

Reply:
Moved down. Uhh stop doing the jquery hack stuff yet? Not yet, I'm going to try to fix that again in a different PR. I spent hours on it before and it was not easy.. maybe now that it's in TS it will be easier to do.